src/main/scala/no/ndla/frontpageapi/model/domain/SubjectFrontPageData.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Part of NDLA frontpage_api.
+ * Copyright (C) 2018 NDLA
+ *
+ * See LICENSE
+ */
+
+package no.ndla.frontpageapi.model.domain
+
+import io.circe.generic.semiauto._
+import io.circe.{Decoder, Encoder, HCursor}
+import io.circe.generic.auto._
+import io.circe.parser._
+import no.ndla.frontpageapi.FrontpageApiProperties
+import scalikejdbc.WrappedResultSet
+import scalikejdbc._
+import cats.implicits._
+
+import scala.util.Try
+
+case class SubjectFrontPageData(id: Option[Long],
+                                displayInTwoColumns: Boolean,
+                                twitter: String,
+                                facebook: String,
+                                bannerImageId: Long,
+                                subjectListLocation: Int,
+                                about: AboutSubject,
+                                topical: SubjectTopical,
+                                mostRead: ArticleCollection,
+                                editorsChoices: ArticleCollection,
+                                latestContent: ArticleCollection)
+
+object SubjectFrontPageData extends SQLSyntaxSupport[SubjectFrontPageData] {
+  override val tableName = "subjectpage"
+  override val schemaName = FrontpageApiProperties.MetaSchema.some
+
+  private def getDecoder(id: Long): Decoder[SubjectFrontPageData] =
```

Comment:
Er dette en custom decoder? Hva er grunnen til at denne må være mer "manuell" enn feks `SubjectPageData`?

Reply:
Det er noe som henger igjen. Begynte med å ha `id` med type `Long` og ikke `Option[Long]`, da måtte jeg ha en egen decoder. Endra det tilbake i etterkant så det er egentlig ikke behov for en custom decoder. Jeg fikser